code/datums/components/riding/riding_mob.dm

```diff
@@ -179,7 +179,6 @@
 	unequip_buckle_inhands(parent)
 	var/mob/living/carbon/human/H = parent
 	H.remove_movespeed_modifier(/datum/movespeed_modifier/human_carry)
-	former_rider.set_density(TRUE)
```

Comment:
From quick glance won't this make piggybacked person nondense until lying down or other density change.

Reply:
I completely forgot that piggybacking was a thing, I now set it to set density depending on your body's position.